drivers/gles2/shader_compiler_gles2.cpp

```diff
@@ -758,11 +758,13 @@ String ShaderCompilerGLES2::_dump_node_code(SL::Node *p_node, int p_level, Gener
 				} break;
 
 				case SL::OP_SELECT_IF: {
+					code += "(";
```

Comment:
Those changes in shader compiler are unrelated to this feature. Please put these in another PR.

Reply:
It seems that I took some random commit in my branch by mistake, will fix

Reply:
i dropped that commit, dunno what happend here